docs/roadmap.md

```diff
@@ -2,10 +2,15 @@
 
 These are our upcoming features:
 
-- Support extracting runtime configurations from additional Cloud providers such as Azure & GCP
-- CIS certifications for k8s and release 1.5
-- Integration with AWS Security Hub
-- Extend coverage of queries for GCP & Azure
+- Release of the v1.6
+- KICS will support the following technologies:
+    - Pulumi
+    - Serverless Framework
+    - Crossplane
+    - Knative
+- Kuberneter (resource data gathering through Kubernetes API)
```

Comment:
Maybe (dynamic gathering and scan of kubernetes cluster data through kubernetes API)